crates/artifact/tests/benchmark.c

```diff
@@ -1,20 +1,15 @@
 #include <stdio.h>
+#include "sightglass.h"
 
 __attribute__((export_name("add")))
-__attribute__((noinline))
-int add(int a, int b) {
+__attribute__((noinline)) int
+add(int a, int b)
+{
     return a + b;
 }
 
-__attribute__((import_module("bench")))
-__attribute__((import_name("start")))
-void bench_start();
-
-__attribute__((import_module("bench")))
-__attribute__((import_name("end")))
-void bench_end();
-
-int main() {
+int main()
```

Comment:
Let's either keep the return type on the same line as the function name or not. Having one function that does that and another that doesn't is inconsistent and makes the code harder to read.

Reply:
Bleh, yeah. I switched IDEs and the new one has weird defaults. (This also reminds me that I should add a Rust lint check in the CI... not sure about a C lint check).